packages/heptagon/heptagon.1.03.01/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "gwenael.delaval@inria.fr"
+authors: "Gwenaël Delaval"
+homepage: "http://bzr.inria.fr"
+dev-repo: "git+ssh://scm.gforge.inria.fr//gitroot/heptagon/heptagon.git"
```

Comment:
The second `//` looks suspicious (but is probably harmless). I can't seem to clone the repo, with any combination of URL modifications. If the repo is not intended to be publicly accessible, please remove this field from the `opam` file.